cypress-tests/cypress/fixtures/create-connector-body.json

```diff
@@ -9,7 +9,7 @@
         "api_key": "api-key",
         "key1": "value1"
     },
-    "test_mode": false,
+    "test_mode": true,
```

Comment:
Wouldn't this affect other connectors? From what I know, this is needed only for specific connectors like `Mollie`?

Reply:
No test_mode should be true only for sandbox